packages/gatsby/src/commands/develop.ts

```diff
@@ -90,14 +96,31 @@ class ControllableScript {
     }
 
     return new Promise(resolve => {
+      if (!this.process) {
+        throw new Error(`Trying to stop process before starting it`)
```

Comment:
Add an article `the`

```suggestion
        throw new Error(`Trying to stop the process before starting it`)
```